application/common/config/main.php

```diff
@@ -53,6 +51,7 @@
 
 $idBrokerConfig = Env::getArrayFromPrefix('ID_BROKER_');
 $idBrokerConfig['validIpRanges'] = Env::getArray('ID_BROKER_validIpRanges');
+$idBrokerConfig['assertValidIp'] = $idBrokerConfig['assertValidBrokerIp'];
```

Comment:
I wonder if we should modify the broker to use the `assertValidBrokerIp ` and not `assertValidIp`.

Reply:
It's not broker configuration. We could modify the Terraform config to pass a different variable name.